apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -115,7 +115,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         VoteStatus status;                                  // Status of the vote
 
         uint256 settingId;                                  // Identification number of the setting applicable to the vote
-        uint256 actionId;                                   // Identification number of the associated disputable action on the attached Agreement
+        uint256 actionId;                                   // Identification number of the associated disputable action on the linked Agreement
```

Comment:
There are a few other places I've used "attached Agreement". Could we search and replace those with "linked Agreement"?